apps/prairielearn/src/pages/authLogin/authLogin.html.js

```diff
@@ -0,0 +1,265 @@
+// @ts-check
```

Comment:
This page is probably quite outdated at this point. I'm going to convert it in another PR, and it should then be obvious what changes are *actually* made in this PR.